src/main/java/core/basesyntax/model/Store.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.model;
+
+public class Store {

```

Comment:
```suggestion
public class FruitTransaction {
```
it's super confusing